.env

```diff
@@ -5,5 +5,5 @@ REACT_APP_DECORATORS=true
 REACT_APP_BABEL_STAGE_0=true
 REACT_APP_CSS_MODULES=true
 OPEN_BROWSER=false
-REACT_APP_DEX_URI=http://localhost:5556
+REACT_APP_DEX_URI=http://192.168.12.35:5556
```

Comment:
Can you revert this back to localhost? We're going to be running dex on the same container as circuit for local dev, so we don't need to hardcode ips anymore.

Reply:
sure